conda-store-server/conda_store_server/server/views/ui.py

```diff
@@ -208,6 +210,7 @@ async def ui_get_build(
             "registry_external_url": server.registry_external_url,
             "entity": entity,
             "spec": yaml.dump(build.specification.spec),
+            "platform": "Windows" if sys.platform == "win32" else "Linux, macOS",
```

Comment:
This needs to be updated because Linux and macOS installers are not compatible: cannot use a Linux installer on macOS